includes/autoloaders.inc.php

```diff
@@ -0,0 +1,73 @@
+<?php 
+
+/**
+ * @file includes/autoloaders.inc.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @ingroup index
+ *
+ * @brief Registers custom autoloaders 
+ */
+
+// registering autoloader function for OJS namespace
+spl_autoload_register(function($class) {
+	$prefix = 'OJS\\';
```

Comment:
in such case, we would need to try to load files from 2 locations: 
- classes/
- lib/pkp/classes 

is that okay?

Also should we search following a specific order?